### PR TITLE
Show bookings

### PR DESCRIPTION
**Pull request by Sajaas**
## PT Link: https://www.pivotaltracker.com/story/show/151365394
-------------------
* Lists users bookings
* Tells user "you have no bookings yet." if they have no bookings

### CODE REVIEW

File: app/views/laundry_rooms/index.html.erb [hunk 1]
```diff
@@ -13,32 +13,45 @@
...
+        <ul>
+          <li><%= bookings[:time].beginning_of_hour.strftime("%A " + "%d/%m/%Y %H:%M") %></li>
+        </ul>
+    <% end %>
+  <% else %>
+   <p>You don't have any bookings yet.</p>
+  <% end %>
+
+  <h1>Book a LaundryTime</h1>
+  <table>
```
Comment (tochman) [inline app/views/laundry_rooms/index.html.erb]: Code formatting feels a bit off here. Please revisit

File: app/views/laundry_rooms/index.html.erb [hunk 2]
```diff
@@ -13,32 +13,45 @@
...
-  <tr>
-    <th>ID</th>
-    <th>Slots</th>
-    <th>Actions</th>
-  </tr>
-  <% @laundry_rooms.each do |room| %>
+<% if current_user %>
+  <h1>Your bookings:</h1>
+    <% if current_user.bookings.any? %>
+    <% current_user.bookings.each do |bookings| %>
```
Comment (tochman) [inline app/views/laundry_rooms/index.html.erb]: The local variable should be named in singular. It represents ONE instance of a booking, right`

File: features/show_bookings.feature [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  Background:
+    Given time is frozen at 2017-09-01
+    And there is a laundry room
+    And there is a user logged in
+
+  Scenario: User sees his laundry time
+    Given user visits the booking page
+    Then I click on "Book" for first "01 Sep 08:00"
+    And I see the text "01/09/2017 08:00"
```
Comment (tochman) [inline features/show_bookings.feature]: Why not format this date as we format dates in Sweden `2017-09-01` And also, perhaps you should stick to one date formatting format throughout the interface.

File: app/views/laundry_rooms/index.html.erb (hunk 2 above)
Comment (Sajaas) [reply app/views/laundry_rooms/index.html.erb]: Oh, good point!

File: features/show_bookings.feature (hunk 3 above)
Comment (Sajaas) [reply features/show_bookings.feature]: Fixing that!

**Review by tochman — APPROVED** [review]
👍🏼

**Pull request merged**